taskcluster/scripts/build/windows_clang_cl.ps1

```diff
@@ -24,9 +24,12 @@ Copy-Item -Path $env:VCToolsRedistDir\\MergeModules\\Microsoft_VC143_CRT_x86.msm
 
 
 # Setup Openssl Import
-$SSL_PATH = resolve-path "$FETCHES_PATH/QT_OUT/SSL"
-$env:OPENSSL_ROOT_DIR = (resolve-path "$SSL_PATH").toString()
-$env:OPENSSL_USE_STATIC_LIBS = "TRUE"
+$SSL_PATH = "$FETCHES_PATH/QT_OUT/SSL"
+if (Test-Path -Path $SSL_PATH) {
```

Comment:
Should we also remove the openssl binaries from the windows\installer\MozillaVPN.wxs?
        <File Source="../../libcrypto-1_1-x64.dll" />
        <File Source="../../libssl-1_1-x64.dll" />

Reply:
Yes, once we will ship with that version. This build's task is to make sure we have build's with the next version of qt ready and passable to qa at any point in time :)